### PR TITLE
even more entity things

### PR DESCRIPTION
**Pull request by eristhea**
when combined with #361, this pr gives the entire `entity` package decent coverage. not perfect, but decent.
I aimed to map the whole package but decided that I didn't want an unreviewable monster

- maps the new `DisplayEntity` and `InteractionEntity`
- misc

### CODE REVIEW

File: mappings/net/minecraft/entity/damage/DamageTracker.mapping [hunk 1]
```diff
@@ -1,4 +1,5 @@
 CLASS net/minecraft/unmapped/C_xkhemiqp net/minecraft/entity/damage/DamageTracker
+	FIELD f_bbxvtwjo INTENTIONAL_GAME_DESIGN_STYLE Lnet/minecraft/unmapped/C_cpwnhism;
```
Comment (EnnuiL) [inline mappings/net/minecraft/entity/damage/DamageTracker.mapping:2]: oh my god i love this so much

File: mappings/net/minecraft/entity/Targeter.mapping [hunk 2]
```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/unmapped/C_vgutocgh net/minecraft/entity/Targeter
```
Comment (EnnuiL) [inline mappings/net/minecraft/entity/Targeter.mapping]: hm, i genuinely wonder if it's "targetter" or "targeter"

**Review by EnnuiL — CHANGES_REQUESTED** [review]
Ah, just one thing!

File: mappings/net/minecraft/entity/Targeter.mapping (hunk 2 above)
Comment (eristhea) [reply mappings/net/minecraft/entity/Targeter.mapping]: getter is the more British way I'd think

File: mappings/net/minecraft/entity/damage/DamageTracker.mapping (hunk 1 above)
Comment (eristhea) [reply mappings/net/minecraft/entity/damage/DamageTracker.mapping:2]: thank mojang!

File: mappings/net/minecraft/entity/Targeter.mapping (hunk 2 above)
Comment (EnnuiL) [reply mappings/net/minecraft/entity/Targeter.mapping]: heck, i'm a bit british then

**Review by EnnuiL — APPROVED** [review]

**Review by OroArmor — APPROVED** [review]

File: mappings/net/minecraft/item/ModelTransformationMode.mapping [hunk 3]
```diff
@@ -0,0 +1,10 @@
+CLASS net/minecraft/unmapped/C_byhguelf net/minecraft/item/ModelTransformationMode
```
Comment (OroArmor) [inline mappings/net/minecraft/item/ModelTransformationMode.mapping]: Can we move this class to `net/minecraft/client/render/model/json`?

**Review by OroArmor — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/item/ModelTransformationMode.mapping (hunk 3 above)
Comment (eristhea) [reply mappings/net/minecraft/item/ModelTransformationMode.mapping]: one concern is that I used the package to signify that it's unique to items, but I like that refactor

**Review by OroArmor — APPROVED** [review]

**Pull request merged**